exercises/strings/replaceCharacter/replaceCharacter.js

```diff
@@ -12,15 +12,47 @@
  * @returns {string} A copy of the input string with target replaced by replaceWith
  */
 
-function replaceCharacter(string, target, replaceWith) {
-  // This is your job. :)
-}
+
+/* arian's plan
+1. obtain string
+2. define the target letter you want to id
+3. iterate through the whole word to find that target letter
+4. store that target letter into something so we can keep count
+3. replace every letter that we targeted from the string 
+4. return that new word variable as a result 
+*/
+
+function replaceCharacter(string, target, replaceWith) {  
+  let newWord=''
+  for (let i=0; i<string.length; i++) {
+    if (string.charAt(i) !== target) {
+      newWord= newWord + string.charAt(i);
+    }
+    else ( newWord = newWord + replaceWith);
+    }
+    return newWord;
+  }
+
+//@jesse this was so fucking hard. i spent 3 hrs on this with the help of lucas // so technically didn't solve  on my own. 
```

Comment:
Start on the array exercises. Those will get you comfortable with iterating over things.

Iterating over the characters in a string and elements of an array are the same.

Reply:
By "start on", I don't mean you should've done those first before you started strings. Rather, do 2-3 from each section at a time.